stories/demos/AllFeatures.tsx

```diff
@@ -56,9 +56,27 @@ function createRows(numberOfRows: number): Row[] {
   return rows;
 }
 
+function isReachedBottom(event: React.UIEvent<HTMLDivElement>): boolean {
```

Comment:
Small grammar fix.
```suggestion
function hasReachedBottom(event: React.UIEvent<HTMLDivElement>): boolean {
```
or
```suggestion
function isAtBottom(event: React.UIEvent<HTMLDivElement>): boolean {
```